src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs

```diff
@@ -225,7 +225,15 @@ public async Task<IActionResult> Edit(AdminNodeEditViewModel model)
             await _adminMenuService.SaveAsync(adminMenu);
 
             await _notifier.SuccessAsync(H["Admin node updated successfully."]);
-            return RedirectToAction(nameof(List), new { id = model.AdminMenuId });
+            if (String.Equals(submit, "saveAndContinue", StringComparison.Ordinal))
```

Comment:
Also the same in the other places.

```suggestion
            if (string.Equals(submit, "saveAndContinue", StringComparison.Ordinal))
```